Sources/TuistCore/Graph/Nodes/TargetNode.swift

```diff
@@ -88,8 +88,9 @@ public class TargetNode: GraphNode {
         dependencies.lazy.compactMap { $0 as? TargetNode }
     }
 
-    public var precompiledDependencies: [PrecompiledNode] {
-        dependencies.lazy.compactMap { $0 as? PrecompiledNode }
+    public var recursivePrecompiledDependencies: [PrecompiledNode] {
```

Comment:
As mentioned below, I think traversing a graph is knowledge that should live in the graph and not in the nodes.

Reply:
I got rid of this method in favour of `findAll(...)` in a `Graph` 👍